react/week1/my-app/src/App.js

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import TodoList from './components/TodoList'
+
+
+const tasks = [
+  {
+    id: 1,
+    task: 'Get out of bed',
+    deadLine: 'Wed Sep 13 2017'
```

Comment:
You can also generate a date like this using the [Date constructor and calling the toDateString method on it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString).

You don't necessarily have to change it here, I just wanted to let you know about it.

Reply:
thanks